plugins/fortune.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+import sys
+import subprocess
+
+if len(sys.argv) < 2:
+    reply = "Missing parameters"
+else:
+    args = sys.argv
+    args.pop(0)
+    args.pop(0)
+
+    cmd = " ".join(args)
+    try:
+        reply = "``` {} ```".format(subprocess.check_output(["fortune"], universal_newlines=True))
+    except subprocess.CalledProcessError:
+        reply = "Something messed up, was it you?"
+try:
+    print(reply)
+except NameError:
```

Comment:
is this not redundant? i dont think you should ever get a name error as no code paths dont assign to `reply`. the other thing is you can also just do ```reply = ""``` at the top to avoid the extra try catch.
try catch are generally considered an expensive operation but in python its actually pretty fine.